keyboards/handwired/leftynumpad/info.json

```
@@ -0,0 +1,36 @@
+{
+    "keyboard_name": "LeftyNumpad",
+    "url": "https://github.com/tomswartz07",
+    "maintainer": "Tom Swartz",
+    "width": 5,
+    "height": 5,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                    {"label":"0,0", "x":0, "y":0},
+                    {"label":"1,0", "x":1, "y":0},
+                    {"label":"2,0", "x":2, "y":0},
+                    {"label":"3,0", "x":3, "y":0},
+                    {"label":"4,0", "x":4, "y":0},
+                    {"label":"0,1", "x":0, "y":1},
+                    {"label":"1,1", "x":1, "y":1},
+                    {"label":"2,1", "x":2, "y":1},
+                    {"label":"3,1", "x":3, "y":1},
+                    {"label":"5,1", "x":4, "y":1, "h":2},
+                    {"label":"0,2", "x":0, "y":2},
+                    {"label":"1,2", "x":1, "y":2},
+                    {"label":"2,2", "x":2, "y":2},
+                    {"label":"3,2", "x":3, "y":2},
```

These objects need to be in the same order that `keymap.c` uses.

```suggestion
                    {"label":"0,2", "x":0, "y":2},
                    {"label":"1,2", "x":1, "y":2},
                    {"label":"2,2", "x":2, "y":2},
                    {"label":"3,2", "x":3, "y":2},
                    {"label":"5,1", "x":4, "y":1, "h":2},
```